hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/ConfigurationKeys.java

```diff
@@ -27,6 +27,15 @@
 @InterfaceAudience.Public
 @InterfaceStability.Evolving
 public final class ConfigurationKeys {
+
+  /**
+   * Each time a FS instance is created a Getacl call is made. If the call
+   * fails with 400 Bad request, the account is determined to be a non-HNS
+   * account.
+   * If this config is present, use that to determine account HNS status. If
+   * config is not present, default behaviour will be calling getAcl.
```

Comment:
Minor. Change comment to :
  /**
   * Config to specify if the configured account is HNS enabled or not. If this config is not set, 
   * getacl call is made on account filesystem root path to determine HNS status.
   */

Reply:
Done